### PR TITLE
Add migration task to switch direction of collection membership

### PR DESCRIPTION
**Pull request by hackmastera**
for upgrade path from sufia 7.3

@projecthydra-labs/hyrax-code-reviewers

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@hackmastera once merged, would you mind adding a reference to this rake task in the 1.0.0 release notes?

**Comment by hackmastera** [comment]
Sure thing.

@mjgiarlo You may want to re-assert approval since I used the nuclear option on rubocop.

**Comment by mjgiarlo** [comment]
@jcoyne I'm inclined to merge. Do you have druthers?

**Comment by jcoyne** [comment]
@mjgiarlo @hackmastera should this merge to the 1-0-stable branch, or should we backport after merging to master?

**Comment by hackmastera** [comment]
It's in cc already, so maybe just 1.0? That would mean people can't use an alternate upgrade path (i.e. must go first to cc2). Not sure if it's worth having in master just in case a cc person wants to do it differently.

File: lib/hyrax/collections_migration.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Hyrax
+  class CollectionsMigration
+    def self.run
+      ::Collection.all.each do |collection|
+        collection.members.each do |member|
+          member.member_of_collections << collection
+          member.save
```
Comment (jcoyne) [inline lib/hyrax/collections_migration.rb]: should we consider save! or checking the return values?

File: lib/hyrax/collections_migration.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        end
+        collection.members = []
+        collection.save
```
Comment (jcoyne) [inline lib/hyrax/collections_migration.rb]: same here.

File: lib/hyrax/collections_migration.rb (hunk 1 above)
Comment (atz) [inline lib/hyrax/collections_migration.rb]: Of course if you throw exception or abort, your repo data is stuck in an unworkable state.  But it is better to make it obvious than silently propagate that error.

**Comment by hackmastera** [comment]
closing to move to 1-0-stable branch

**Pull request closed without merging**